README.md

```diff
@@ -1,2 +1,56 @@
 # tidb-ctl
-TiDB Controller
+
+TiDB Controller (tidb-ctl) is a command line tool for TiDB Server (tidb-server).
+
+## Build
+
+1. Make sure [*Go*](https://golang.org/) (version 1.9+) is installed.
+1. Use `make` in repo root path. `tidb-ctl` will build in `bin` directory.
```

Comment:
will build in -> will be in

Reply:
README.md will be generated by cobra, it will totally rewrite in next pull request.

Reply:
PTAL